File: python/sequentialSearch.py
def sequential_search(element, list):
  for i in range(len(list)):
    if list[i] == element:
      return i
  return -1

element = 2
list = [1, 2, 5, 4, 6]

try:
  index = sequential_search(element, list)
  out = (
    f"Element {element} was found in the index {index}"
    if index != -1 else
    "Element was not found"
  )
  print(out)
except Exception as e:
  print(f"Error: {e}")

File: js/sequentialSearch.js
const element = 5;
const arr = [1, 3, 4, 6, 3];
const sequentialSearch = (element, arr) => {
  for (let i = 0; i < arr.length; i++) {
    if (element === arr[i]) {
      return i;
    }
  }
  return -1;
}

try {
  const index = sequentialSearch(element, arr);
  const out = index != -1
    ? `Element ${element} was found in the index ${index}`
    : 'Element was not found';
  console.log(out);
} catch (error) {
  console.log(error.message);
}
